modules/selenium/src/test/java/org/testcontainers/junit/ChromeRecordingWebDriverContainerTest.java

```diff
@@ -11,23 +13,43 @@
 
 import java.io.File;
 import java.util.Optional;
-
+import static org.junit.Assert.assertTrue;
 import static org.testcontainers.containers.BrowserWebDriverContainer.VncRecordingMode.RECORD_ALL;
 
 @RunWith(Enclosed.class)
 public class ChromeRecordingWebDriverContainerTest extends BaseWebDriverContainerTest {
 
     public static class ChromeThatRecordsAllTests {
 
+        public File vncRecordingDirectory = new File("./build/");
```

Comment:
Thanks for adding a test!

Could you please create a temp directory here instead of reusing `./build/`?
It will also help simplifying the assertion (no need to randomize the test name)

Reply:
Done...